components/toast/toast.stories.js

```diff
@@ -130,6 +130,14 @@ export const Default = {
     title: 'Base title (optional)',
     kind: 'base',
   },
+
+  parameters: {
+    percy: {
+      args: {
+        show: true,
+      },
+    },
+  },
```

Comment:
Should we just make this the default for all stories and then set it to false on the ones we don't want? You can set parameters globally in `preview.js`

Reply:
It's possible, but we are setting different variables on different stories so it's not completely generic.

Reply:
Oh I see it's the actual "show" prop. I thought this was a built in Percy attribute that enables whether to do visual testing or not